test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -313,16 +313,23 @@ public async Task FunctionDispatcher_DoNot_Restart_ErroredChannels_If_WorkerRunt
         }
 
         [Theory]
-        [InlineData(@"node", false, true, true)]
-        [InlineData(@"node", true, false, true)]
-        [InlineData(@"node", true, true, true)]
-        [InlineData(@"node", false, false, false)]
-        [InlineData(@"java", false, true, false)]
-        public async Task FunctionDispatcher_ShouldRestartChannel_Returns_True(string language, bool isWebHostChannel, bool isJobHostChannel, bool expectedResult)
+        [InlineData(@"node", @"node", false, true, true)]
+        [InlineData(@"node", @"node", true, false, true)]
+        [InlineData(@"node", @"node", true, true, true)]
+        [InlineData(@"node", @"node", false, false, false)]
+        [InlineData(@"node", @"java", false, true, false)]
+        [InlineData(@"node", "", false, true, false)]
+        [InlineData("", "", false, true, false)]
+        [InlineData("", @"node", false, true, false)]
+        [InlineData("", @"node", true, false, false)]
+        [InlineData("", @"node", true, true, false)]
+        [InlineData("", @"node", false, false, false)]
+        [InlineData("", @"java", false, true, false)]
```

Comment:
nit: add case for null workerRunime. Also, remove @ for strings :)

Reply:
@pragnagopa - removed @! also - removing the "" and using null instead, as _workerRuntime will always either be a value or will be null and not be "".